tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/services/ClientResourceAccessReturnTest.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Let's rename this file to `ResourceMethodReturnTypeTest`

Reply:
Updated 2504be25de6173179816f9cda84efa8d7ab3883f